plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioParquetConnectorTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import io.trino.Session;
+import org.apache.iceberg.FileFormat;
+
+public class TestIcebergMinioParquetConnectorTest
+        extends BaseIcebergMinioConnectorTest
+{
+    public TestIcebergMinioParquetConnectorTest()
+    {
+        super(FileFormat.PARQUET);
+    }
+
+    @Override
+    protected boolean supportsIcebergFileStatistics(String typeName)
+    {
+        return true;
+    }
+
+    @Override
+    protected boolean supportsRowGroupStatistics(String typeName)
+    {
+        return !(typeName.equalsIgnoreCase("varbinary") ||
```

Comment:
Can you please use a utility method to avoid code duplication in: 
- io.trino.plugin.iceberg.TestIcebergParquetConnectorTest#supportsRowGroupStatistics
- io.trino.plugin.iceberg.TestIcebergMinioParquetConnectorTest#supportsRowGroupStatistics

Same comment for `withSmallRowGroups`

Reply:
Will add base class that will catch both cases.